docs/sources/tempo/traceql/metrics-queries/functions.md

```diff
@@ -181,21 +196,51 @@ To demonstrate this flexibility, consider this nonsensical quantile on `span.htt
 { name = "GET /:endpoint" } | quantile_over_time(span.http.status_code, .99, .9, .5)
 ```
 
-This computes the 99th, 90th, and 50th percentile of the values of the `status_code` attribute for all spans named `GET /:endpoint`.
+This computes the 99th, 90th, and 50th percentile of the values of the `status_code` attribute for all spans named
+`GET /:endpoint`.
 This is unlikely to tell you anything useful (what does a median status code of `347` mean?), but it works.
 
 As a further example, imagine a custom attribute like `span.temperature`.
-You could use a similar query to know what the 50th percentile and 95th percentile temperatures were across all your spans.
+You could use a similar query to know what the 50th percentile and 95th percentile temperatures were across all your
+spans.
+
+## `topk` and `bottomk` functions
+
+TraceQL supports the `topk` and `bottomk` functions that let you aggregate and process TraceQL metrics.
+These functions are similar to their equivalent PromQL functions. For example:
+
+- `{ } | rate() by(resource.service.name) | bottomk(5)`
+- `{ } | rate() by(resource.service.name) | topk(5)`
+
+When a query response is larger than the maximum, you can use to return only the specified number
+from 1 through `k` of the number of the top or bottom results.
```

Comment:
I think we have a missing word in here: "you can use to return..." 
What word should be here?